### ISSUE TITLE
VAN _unpack_signups() method errors on empty table

### ISSUE THREAD

**Issue by ydamit** [move-coop/parsons#498]
The `_unpack_signups()` internal method is frequently used in the `Signups()` VAN class, but it throws a hard-to-trace `ValueError` when the method using it returns an empty table, ultimately settling on the message `'person' is not in list`.

I propose that the easiest solution would be to add an `if` statement to the beginning of `_unpack_signups()` to check for rows > 0

**Comment by JackHouk** [comment]
This seems like the issue is really with `unpack_dict()` in `etl`.

If we want to allow an "unpack" on columns that don't exist (which is what users would see from the functions that use `_unpack_signups()`) then wouldn't it be preferable for `unpack_dict()` to allow it so it's consistent across modules?

Alternatively, for consistency, if we don't want to allow unpacking nonexistent columns then we should throw a clearer error when `unpack_dict()` is called on a nonexistent column. Even if all we do is raise the same exception as petl from `etl.unpack_dict()` it will at least make the stack trace much clearer.

Basically, we could put something like this into `etl.unpack_dict()`
```
if column not in self.table.columns():
            return self
            # Alternatively, raising an error here cleans up the stack trace

            # raise ValueError(f'Column name \'{column}\' not found')
```

**Comment by shaunagm** [comment]
I don't think the proposed change was made, but pinging @SorenSpicknall again to confirm
